agent/proxycfg/state.go

```diff
@@ -32,6 +32,8 @@ const (
 	serviceResolversWatchID            = "service-resolvers"
 	gatewayServicesWatchID             = "gateway-services"
 	gatewayConfigWatchID               = "gateway-config"
+	inlineCertificateConfigWatchID     = "inline-certificate-config"
+	routeConfigWatchID                 = "route-config"
```

Comment:
These prefixes are used when subscribing to config entry changes and then passed along to the change handler. In the change handler, this prefix indicates what sort of change happened so that we can respond to it